src/main/resources/templates/index.ftl

```diff
@@ -79,6 +79,9 @@
                                     </#if>
                                 </#list>
                             </td>
+                            <td>
+                                <span>${user.getStarNumber() / 2}</span>
```

Comment:
Are we sure this renders well when the user has an uneven number of stars ?

Reply:
If the question is : does it show 0 when people doesn't have done any challenge, the answer is yes, it shows 0.